### ISSUE TITLE
[Request] F-Droid availability 

### ISSUE THREAD

**Issue by glocalglocal** [mickstar/Zoo-For-Zotero#88]
Can the app be made available through F-Droid as well? Some users can't or won't get apps from Google Play.

**Comment by mickstar** [comment]
Yes, i would like that as well. I made the necessary code changes I just have to submit it to F-Droid to be compiled.
It's a little cumbersome which is why I haven't finished doing it, but I will get around to it eventually.

**Comment by lightnin** [comment]
If it's still possible it would be great if your app was available on F-Droid. With the changes in annotations / highlighting in the recent major release of Zotero, many folks like me will be looking for open-source apps to read and sync our Zotero docs and notes across e-reader devices, and it looks like this one is one of the best. F-Droid is already on my Onyx Boox Nova2, and I'd prefer to keep the app store off.  Many thanks for all your work making Zoo!

**Comment by Hillside502** [comment]
> recent major release of Zotero

The last release was 12 months ago
https://github.com/mickstar/Zoo-For-Zotero/releases

**Comment by lightnin** [comment]
> > recent major release of Zotero
> 
> The last release was 12 months ago https://github.com/mickstar/Zoo-For-Zotero/releases

Of Zotero. Just Zotero, not Zoo-For-Zotero.

**Comment by Hillside502** [comment]
> > > recent major release of Zotero
> > 
> > 
> > The last release was 12 months ago https://github.com/mickstar/Zoo-For-Zotero/releases
> 
> Of Zotero. Just Zotero, not Zoo-For-Zotero.

Corrected!

**Comment by aadityabhatia** [comment]
This appears to be a duplicate of #39, which was closed as "wontfix".

**Comment by IzzySoft** [comment]
@mickstar if you're still interested: there's an [inclusion request](https://gitlab.com/fdroid/rfp/-/issues/2251) open at F-Droid, which is currently stuck. Details in the report there, but the basic issues are [non-free dependencies](https://github.com/mickstar/Zoo-For-Zotero/blob/master/app/build.gradle#L102) (Crashlytics, Firebase) where a foss-flavor coming without them would help – and missing Fastlane structures with the app's metadata (description, screenshots etc – be welcome to use my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688) for assistance with that).

Any chance that could be tackled? Thanks in advance!

**Comment by IzzySoft** [comment]
@mickstar without he issue being addressed (and solved), we cannot process the inclusion request and simply have to close it: F-Droid doesn't permit proprietary components being included. Seeing that there wasn't any commit since 10/2022 (and the last release was made in 2021), is the app even maintained still?

The RFP will now enter the closure process: if nothing happens, it will be closed in about a month as we cannot process it.

**Comment by Hillside502** [comment]
@IzzySoft @mickstar 
Abandonware by any other name!
